ifrs17/DataModel/DataStructure.ipynb

```diff
@@ -2154,6 +2155,115 @@
             "metadata": {},
             "execution_count": 0,
             "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Variable for Recording the Logs"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "The following variables are defined in order to keep the file information and keep trace of the logs, each time we access the DB."
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Variable that Tracks the DB Activity"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public record KeyedActivity : KeyedRecord",
+                "\n{",
+                "\n    public string UserName {get; init;}",
+                "\n",
+                "\n    public DateTime StartDateTime {get; init;}",
+                "\n",
+                "\n    public DateTime EndDateTime {get; init;}",
+                "\n",
+                "\n    public ActivityLogStatus Status {get; init;}",
+                "\n",
+                "\n    public string Category {get; init;}",
+                "\n",
+                "\n    public string Format {get; init;}",
```

Comment:
What is Format?

Reply:
One of the formats that we have, e.g. DataNodeState, Cashflows, Actual, etc.